mlir-tensorrt/python/mlir_tensorrt_compiler/pyproject.toml

```diff
@@ -26,5 +26,6 @@ dependencies = [
     "pybind11>=2.8.0, <=2.10.3",
     "PyYAML>= 5.3.1, <=6.0.1",
     "dataclasses>=0.6, <=0.8",
+    "tensorrt~=10.0",
```

Comment:
This is not true. You can build the project to target TensorRT 8.6. We shouldn't bake `10.0` into the requirements like this.

Reply:
@christopherbate should we specify any version range or leave it completely open? Maybe `>= 8.6`?